examples/fibonacci.rs

```diff
@@ -95,7 +100,7 @@ fn fibo_circuit<F: Field + From<u64> + Hash>() -> (Circuit<F>, Option<Assignment
             let mut a = 1;
             let mut b = 2;
 
-            for _i in 1..11 {
+            for _i in 1..16 {
```

Comment:
Hyperplonk requires 2^integer number of rows, but in practice this can be replaced by padding, so this shouldn't affect our example.